tests/unit/core/test_addonManager.js

```diff
@@ -197,23 +197,27 @@ describe(__filename, () => {
       );
     });
 
-    it('rejects if the install returns an error', () => {
-      const error = new Error('oops');
+    // See: https://github.com/mozilla/addons-frontend/issues/8633
+    it('logs and ignores rejected install errors', async () => {
+      let finishInstall;
+      const installToFinish = new Promise((resolve) => {
+        finishInstall = resolve;
+      });
+      const _log = getFakeLogger();
+
       fakeInstallObj.install = sinon.spy(() => {
-        return Promise.reject(error);
+        return Promise.reject(new Error('oops'));
       });
 
-      return (
-        addonManager
-          .install(fakeInstallUrl, fakeCallback, {
-            _mozAddonManager: fakeMozAddonManager,
-            src: 'home',
-          })
-          // The second argument is the reject function.
-          .then(unexpectedSuccess, () => {
-            sinon.assert.calledOnce(fakeInstallObj.install);
-          })
-      );
+      addonManager.install(fakeInstallUrl, fakeCallback, {
+        _log,
+        _mozAddonManager: fakeMozAddonManager,
+        onIgnoredRejection: () => finishInstall(),
+        src: 'home',
+      });
```

Comment:
Shouldn't this still include the `catch()` you added previously? I'm thinking of a case where something inside `_mozAddonManager.createInstall().then(...)` throws unexpectedly.

Reply:
I could not easily come up with a failing test case.

If `_mozAddonManager.createInstall().then(...)`, then it will be caught in `installAddon` I think.